Btms.Business/Services/Decisions/Finders/ChedPPDecisionFinder.cs

```diff
@@ -3,29 +3,22 @@
 namespace Btms.Business.Services.Decisions.Finders;
 
 // ReSharper disable once InconsistentNaming
-public class ChedPpPhsiDecisionFinder : IDecisionFinder
+public class ChedPPDecisionFinder : IDecisionFinder
 {
-    public const string CheckCode = "H219";
+    public static readonly string[] CheckCodes = ["H218", "H220", "H219"];
```

Comment:
Should/can these be re-using the info from ImportNotificationTypeEnumExtensions ? Rather than have multiple lists?